### PR TITLE
Kick-off for the dat-ecosystem organization

### PR DESCRIPTION
**Pull request by martinheidegger**
> Let's get this started!

Hello Friends, let's kick-off the Dat Ecosystem organization! 🚀 

This PR contains the first official version of the [📃 Manifesto](https://github.com/dat-ecosystem/organization/blob/kickoff/MANIFESTO.md), the [📃 Code of Conduct](https://github.com/dat-ecosystem/organization/blob/kickoff/code-of-conduct.md) and the [📃  CoC Reporting Guide](https://github.com/dat-ecosystem/organization/blob/kickoff/code-of-conduct-contact.md). They are copies of documents previous shared at [🔗 HackMD](https://hackmd.io/XAfk8Q5NQJiCHYpB5vjZow?view).

My thinking is that this manifest will take effect as soon as **all consortium members approved this** PR - making it official! 🥳

### CODE REVIEW

File: README.md
Comment (cblgh) [inline README.md]:
```suggestion
The "Dat Ecosystem" is a project composed of people interested in things like the [hypercore-protocol](https://hypercore-protocol.org).
```

Comment (martinheidegger) [reply README.md]: thanks!

File: README.md
Comment (cblgh) [inline README.md]:
```suggestion
You can meet us at [🗓 Events](https://events.dat.foundation/), in the [💬 Discord Chat](https://discord.gg/fZymdRfctR), at [👥 Comm-Comm casual meetups](https://comm-comm.datproject.org/) or [💬 Cabal](https://github.com/dat-ecosystem/dat-ecosystem.github.io/blob/main/README.md#connect-to-cabal-with-cli-or-download-cabal-desktop).
```

File: code-of-conduct-contact.md
Comment (cblgh) [inline code-of-conduct-contact.md]: is this email registered? or will emails there bounce? i think it would be best to register it before this passes, or remove this line and direct people to the discord

Comment (martinheidegger) [reply code-of-conduct-contact.md]: It will be registered before this PR gets merged afaik. cc. @ninabreznik (who registered the domain)

File: code-of-conduct-contact.md
Comment (cblgh) [inline code-of-conduct-contact.md]:
```suggestion
We would much rather have a few extra reports where we decide to take no action, than miss a report of an actual violation.
```

Comment (martinheidegger) [reply code-of-conduct-contact.md]: ... ! ... Thanks!

File: code-of-conduct-contact.md
Comment (cblgh) [inline code-of-conduct-contact.md]: dat consortium? not dat ecosystem? this might be old

also: what is the proper capitalisation of the group? i think it would be good to write somewhere & be consistent with in official documents :)

File: code-of-conduct-contact.md
Comment (cblgh) [inline code-of-conduct-contact.md:60]: committee? or working group?

File: code-of-conduct-contact.md
Comment (cblgh) [reply code-of-conduct-contact.md]: ah just saw this above
>  This group is **governed by individuals** that form together _"the Consortium"_. The Consortium works to:

perhaps "the Consortium" would be better here, then.

Comment (martinheidegger) [reply code-of-conduct-contact.md]: The mail will be from the consortium of "Dat Ecosystem". Updated in the last commit. Thanks for noticing!

File: code-of-conduct.md
Comment (cblgh) [inline code-of-conduct.md:38]: 👍 i love positively framed community documents

File: code-of-conduct-contact.md
Comment (cblgh) [reply code-of-conduct-contact.md]: 👍 (emoji in response to martin's comment above!)

**Review by cblgh — APPROVED** [review]
LGTM! Thank you everyone for your tireless work!

File: code-of-conduct-contact.md
Comment (martinheidegger) [reply code-of-conduct-contact.md:60]: In my thinking: You send it to the working group's email but the entire consortium (committee?) will be able to see it.

File: code-of-conduct-contact.md
Comment (ninabreznik) [reply code-of-conduct-contact.md]: I just posted a question regarding the email in the discord, please take a look and respond once you'll have time. :)

File: code-of-conduct-contact.md
Comment (ninabreznik) [reply code-of-conduct-contact.md:60]: Maybe comittee sounds to rigid and working group could be used as a synonym. That's at least how I understood the comment.

**Comment by martinheidegger** [comment]
Hello Friends, I just updated the PR to use https://dat-ecosystem.org/organization/ instead of emails for private contact.

_I shared the private key for the consortium in the consortium member signal group (which self-deletes after 5 minutes, ask again if you didn't catch it). Using the private key and the helper homepage, you should be able to follow up on #2._

**Review by Frando — APPROVED** [review]

**Review by cblgh — APPROVED** [review]

**Review by dpaez — APPROVED** [review]

**Review by ninabreznik — APPROVED** [review]

**Review by pfrazee — APPROVED** [review]

**Review by RangerMauve — APPROVED** [review]

**Review by zootella — APPROVED** [review]

**Review by serapath — CHANGES_REQUESTED** [review]
Thanks @martinheidegger 
Also thank you so much for the cool messaging tool we can use for the code of conduct :-)

There is another document called `code-of-conduct-contact.md` which is new to me and maybe you could just make a second pull request for it so we can discuss that? Is it even needed to define those things in detail? Not sure if people maybe skipped that or if they already read through, but if i'm not the only one who obviously missed it so far, then maybe it would be good to go trough it once during the consortium or at least make it an explicit "homework" to see if folks agree with it.


Otherwise, I agree with the **manifesto** and the **code of conduct** and approve them so we can merge those for sure :-)

**Comment by martinheidegger** [comment]
@serapath The reporting guide was also linked in the markdown files. For the Code of Conduct to work the reporting guide is needed. Can you take a look at it?

File: code-of-conduct.md
Comment (serapath) [inline code-of-conduct.md:17]: ok, so i could probably live with a version that removes the following to paragraphs:

> 1. What if your report concerns a possible violation by a consortium member?
> 2. What happens after you file a report?

To me they feel like too much of an involved process with fine grained rules and even time guarantees and promise of "high priority" and stuff like that.

**and instead replaces them with a line that says something like:**

*"we will do the best we can and use our common sense and try to be lovely people and good people and be fair and find a way to resolve it...."*  or something in that spirit.


And once we have an actual real world "incident" (hopefully never) we handle it with best intentions and common sense and if there are problems, then use that experience to add guarantees or process details or whatever.

**Review by serapath — CHANGES_REQUESTED** [review]

File: code-of-conduct.md
Comment (martinheidegger) [reply code-of-conduct.md:17]: Okay, I will propose some changes to the reporting guide within a day.

Comment (martinheidegger) [reply code-of-conduct.md:17]: Merged #4 is it okay to you now? @serapath

**Review by serapath — APPROVED** [review]

**Comment by martinheidegger** [comment]
🎉  Happy Birthday!

**Pull request merged**
